src/lib.rs

```diff
@@ -1047,10 +1047,12 @@ impl<'ctx> Bindings<'ctx> {
             if !has_target_arg {
                 // TODO: distinguish C and C++ paths? C++'s should be enough, I
                 // guess.
-                for path in clang.cpp_search_paths.into_iter() {
-                    if let Ok(path) = path.into_os_string().into_string() {
-                        options.clang_args.push("-isystem".to_owned());
-                        options.clang_args.push(path);
+                if let Some(cpp_search_paths) = clang.cpp_search_paths {
+                    for path in cpp_search_paths.into_iter() {
```

Comment:
This already existed, but the `into_iter` isn't necessary right? It should be automatically called by a `for` loop, correct?